transactor/README.md

```diff
@@ -49,48 +61,64 @@ cd ~/git/wormbase-architecture
 source wb-cf-transactors/bin/activate
 ```
 
-and optionally setup an alias to the command (or add to `$PATH`):
+### Manage command
+
+The following executable is used to manage transactors:
 
 ```bash
-DATOMIC_VERSION="0.9.5561.56"
+bin/manage
 ```
 
-Adding `--help` to the end of the aliased command above will describe
-the available options and any required arguments.
+This command expects two positional arguments (SETTINGS and CF_STACK_NAME),
+followed by a sub-command, followed by more positional and optional arguments.
 
-### CloudFormation stack operations for managing the datomic transactor
-The stack was created the _very first_ time using the datomic tools'
-_create-cf-stack_ command.
+Adding `--help` to the end of the manage command (and its sub
+commands) will describe the available options and any required
+arguments.
 
-Assuming this repository is checked-out to the home directory of the
-current user.
+Transactors may managed for more than one project.
+(Currently  web production and the "names" projects).
 
-In the examples below:
+Each project should have its own settings file in the `./config` directory,
+detailing specific instance type and datomic memory requirements.
 
-  * `$WS_RELEASE` should be the name of the data release (and DynamoDB
-    table) that you wish to use.
+### Examples of running the "create" command for each project
 
-  * `$DESIRED_CAPACITY` should be the number of transactors desired to
-    be in service (Currently this is permitted to be 1 or 2).
+### CloudFormation stack operations for managing the datomic transactor
+Assuming this repository is checked-out to the home directory of the
+current user.
 
-#### Creating a new datomic transactor CloudFormation stack
+#### Main WormBase Migration DB
 
 ```bash
-cd transactor;
-./bin/manage --cf-stack-name WBTransactor<db version> create "${WS_RELEASE}" "${DATOMIC_VERSION}"
+DDB_TABLE="WS265"
+CF_STACK_NAME="WBTransactorWS265"
+DATOMIC_VERSION="0.9.5697"
+DESIRED_CAPACITY=1
+SETTINGS="config/web-prod-params.json"
+
+bin/manage $SETTINGS $CF_STACK_NAME \
+           create  \
+           --desired-capacity $DESIRED_CAPACITY \
+           $DDB_TABLE \
+           $DATOMIC_VERISON
 ```
 
-#### Updating an existing datomic transactor CloudFormation stack
-
-Currently, you can update any combination of the options available,
-but at least one of `$WS_RELEASE`, `$DESIRED_CAPACITY` or
-`$DATOMIC_VERSION` must be supplied for an update command to proceed.
+#### Names DB
 
-For example, to update the desired capacity to `1` (assuming it is
-currently set to `2`):
 
 ```bash
-cf-transactors --cf-stack-name WBTransactor<db version> update --desired-capacity=1
+DDB_TABLE="WSNames"
+CF_STACK_NAME="WBNamesTransactor"
+DATOMIC_VERSION="0.9.5697"
+DESIRED_CAPACITY=2
+SETTINGS="config/web-prod-params.json"
+
+bin/manage $SETTINGS $CF_STACK_NAME \
+           create  \
```

Comment:
Should this say update instead of create?

Reply:
I removed the update examples for brevity, should be as is.